src/common/redis_module/ray_redis_module.cc

```diff
@@ -419,6 +419,8 @@ bool PublishObjectNotification(RedisModuleCtx *ctx,
 
   RedisModuleCallReply *reply;
   reply = RedisModule_Call(ctx, "PUBLISH", "ss", channel_name, payload);
+  auto num_clients = RedisModule_CallReplyInteger(reply);
+  CHECKM(num_clients > 0, "Published to %lld clients", num_clients);
```

Comment:
Let's not actually commit this check. I think this check is a really good idea, but this could mean that publishing to an object manager that has died will kill Redis.